src/App.js

```diff
@@ -1,26 +1,19 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { Switch, Route } from "react-router-dom";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import "./App.css";
+import CommunityStatisticsPage from "./pages/community-stats/community-statistics.component";
+
+class App extends React.Component {
+  render() {
+    return (
+      <div>
```

Comment:
You can use a [React Fragment](https://reactjs.org/docs/fragments.html) here instead of a `div`, like this
```jsx
<>
<Switch>...</Switch>
</>
```